feconf.py

```diff
@@ -645,6 +645,8 @@ def get_empty_ratings():
 QUESTIONS_URL_PREFIX = '/question_player_handler'
 RECENT_COMMITS_DATA_URL = '/recentcommitshandler/recent_commits'
 RECENT_FEEDBACK_MESSAGES_DATA_URL = '/recent_feedback_messages'
+REVIEW_TEST_URL_PREFIX = '/review_test'
+REVIEW_TEST_DATA_URL_PREFIX = '/review_test/data'
```

Comment:
Currently I think the convention is /reviewtesthandler/data or /review_test_handler/data (see other URLs)

Reply:
Done.